Makefile

```diff
@@ -0,0 +1,17 @@
+override RUNTIME_IMAGE ?= golang:1.8-alpine
+
+docker-test:
+	docker build -f Dockerfile.test -t faunadb-go-test:latest --build-arg RUNTIME_IMAGE=$(RUNTIME_IMAGE) .
+ifdef FAUNA_ROOT_KEY
+ifdef FAUNA_ENDPOINT
```

Comment:
Instead of add those `ifdef` inside the target and repeat the docker command I would assign a variable before and just use that variable

Reply:
Thanks. This is actually my first time actually doing anything non-trivial in make.